### ISSUE TITLE
Can't build android after upgrading to ExpoKit 35

### ISSUE THREAD

**Issue by kuzkokov** [expo/expo#5747]
### Environment

 Expo CLI 3.0.10 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.15.3 - /usr/local/bin/node
      Yarn: 1.15.2 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.5 AI-191.8026.42.35.5791312
      Xcode: 11.0/11A420a - /usr/bin/xcodebuild
    npmPackages:
      expo: ^35.0.0 => 35.0.0 
      react: 16.8.3 => 16.8.3 
      react-native: https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz => 0.59.8 
      react-navigation: ^2.18.3 => 2.18.3 
    npmGlobalPackages:
      expo-cli: 3.0.10

Android, ExpoKit

### Steps to Reproduce

Hey, guys! I just upgrade my ExpoKit app from 34 version to 35. There are not so many changes, and I didn't expect any problems, but here we are.
So, basically, I did the following changes:
from:
```
   "expo": "^34.0.1",
    "expo-document-picker": "^6.0.0",
    "expo-font": "^6.0.1",
    "expo-google-app-auth": "^6.0.0",
    "expo-google-sign-in": "^6.0.0",
    "expo-image-picker": "^6.0.0",
    "expo-permissions": "^6.0.0",
    "expokit": "^34.0.2",
    "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz",
```
to:
```
    "expo": "^35.0.0",
    "expo-document-picker": "^7.0.0",
    "expo-font": "^7.0.0",
    "expo-google-sign-in": "^7.0.0",
    "expo-image-picker": "^7.0.0",
    "expo-permissions": "^7.0.0",
    "expokit": "35.0.0",
    "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
```

and from:
```
  implementation('host.exp.exponent:expoview:34.0.0@aar') {
    transitive = true
    exclude group: 'com.squareup.okhttp3', module: 'okhttp'
    exclude group: 'com.squareup.okhttp3', module: 'okhttp-urlconnection'
  }
```

to:
```
  implementation('host.exp.exponent:expoview:35.0.0@aar') {
    transitive = true
    exclude group: 'com.squareup.okhttp3', module: 'okhttp'
    exclude group: 'com.squareup.okhttp3', module: 'okhttp-urlconnection'
  }
```

And suddenly I start getting [several errors](https://pasteboard.co/IyMkFPA.png). Do you have some ideas, why could it be?
Before versions update, everything was working fine. 🤷‍♂

**Comment by rphlmr** [comment]
I had compilation errors when I migrate (but i'm using managed expo).
My error was to forgot to update "sdkVersion" in my app.json.
I don't know if you have to do it with expo kit :s

**Comment by kuzkokov** [comment]
hi, @rphlmr , thanks for the answer, but I did update `sdkVersion` in the `app.json` as well.
actually, the problem was solved by #5749 -- I just had to remove some libraries from `app/build.gradle`. so I am gonna close it.
